source/clear-linux/guides/telemetrics/telem-guide.rst

```diff
@@ -0,0 +1,566 @@
+
+   .. note::
+
+      To opt-in but not immediately start telemetry services, you will need to run the command :command:`sudo telemctl stop` after the :command:`opt-in` command is entered. Once you are ready to start the service, enter the command :command:`sudo telemctl start`.
+
+#. Opt out of telemetry
+
+   To stop sending telemetrics data from your system, opt out of the telemetry service:
+
+   .. code-block:: bash
+
+      sudo telemctl opt-out
+
+   This creates the file :file:`/etc/telemetrics/opt-out` and stops the telemetry services.
+
+
+Saving Data Locally
+===================
+
+This example requires |CL| to be installed, and telemetry to be enabled on the system.
+
+To change how records are managed, we will be copying the default :file:`/usr/share/defaults/telemetrics/telemetrics.conf` file to :file:`/etc/telemetrics/telemetrics.conf` and editing it.  The changes in the :file:`/etc/telemetrics/telemetrics.conf` file will override the defaults in the :file:`/usr/share/defaults/telemetrics/telemetrics.conf` file. You may need ``root`` permissions to create and edit files in :file:`/etc`. For each example, and for any time you make changes to the configuration file, you will need to restart the client daemons to pick up the changes:
+
+.. code-block:: bash
+
+  sudo telemctl restart
+
+
+The :command:`telemctl journal` command gives you access to features and options of the telemetry journal to assist with system analytics and debug. :command:`telemctl journal` has a number of options to help filter records. Use :command:`-h` or :command:`--help` to view usage options.
+
+
+#. Keep local copy and send records to backend server
+
+   To keep a local copy of the telemetry record and also send it on to the backend server, we will need to change the :guilabel:`record_retention_enabled` configuration key value to :guilabel:`true`.
+
+
+#. Keep all records  -- don't send to backend server
+
+   To keep records on the system without sending them to a backend server, set the :guilabel:`record_server_delivery_enabled` key value to :guilabel:`false`.  Note that you will also need to ensure the the :guilabel:`record_retention_enabled` configuration key value is set to :guilabel:`true` or the system will not keep local copies.
+
+#. Keep and send records to custom server
+
+   This assumes you have set up a custom server according to the next example.
+
+   The server is identified by the :guilabel:`server` setting, and by default records will be sent to the |CL| server :guilabel:`server=https://clr.telemetry.intel.com/v2/collector`.  To change this, you can use an IP address or fully qualified domain name.
+
+
+
+Set up a backend server to collect telemetry records
+====================================================
+
+For this example, start with a clean installation of |CL| on a new system using the :ref:`bare-metal-install` getting started guide and:
```

Comment:
for this example, you'll also need to have them get the git software bundle and possibly python although I think the script downloads all required bundles (other than git)